### DISCUSSION TITLE
Deprecating Learning Lab & Introducing GitHub Skills

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#28323] [Announcements]
No matter where you are on your GitHub journey, there is always more to learn and explore. If you’re new to GitHub, you might want to find out how to get started. If you’re a seasoned GitHub user, you may need to stay up-to-date with what’s new with GitHub’s rich platform.

We launched [GitHub Skills](https://github.blog/2022-06-06-introducing-github-skills/) in June, with popular courses, migrated over from Learning Lab. In doing so, **Learning Lab will shut down on September 1, 2022**. For the latest content and updated learning experience, we encourage you to transition to GitHub Skills now.

Ready to start learning?? Head over to [GitHub Skills](https://skills.github.com/) and dive into interactive courses that are backed by GitHub Actions and designed for everyone. :rocket:

<img width="1155" alt="Screen Shot 2022-08-04 at 8 38 47 AM" src="https://user-images.githubusercontent.com/75342807/182888522-d70ff1fb-0170-4ef3-8803-bf51d3d0f425.png">

**Comment by samus-aran** [comment]
As part of the ongoing housekeeping of GitHub Community Discussions we are scheduling deletion of some discussions that covered the topic of Learning Lab by **Monday 10th October 2022**. This will not affect newer discussions on the deprecation of Learning Lab.

Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
